connect/mirror/src/test/java/org/apache/kafka/connect/mirror/integration/DedicatedMirrorIntegrationTest.java

```diff
@@ -319,11 +397,14 @@ private void writeToTopic(EmbeddedKafkaCluster cluster, String topic, int numMes
             cluster.produce(topic, Integer.toString(i));
         }
     }
-
     private void awaitMirrorMakerStart(final MirrorMaker mm, final SourceAndTarget sourceAndTarget) throws InterruptedException {
+        awaitMirrorMakerStart(mm, sourceAndTarget, CONNECTOR_CLASSES);
+    }
+
+    private void awaitMirrorMakerStart(final MirrorMaker mm, final SourceAndTarget sourceAndTarget, final  List<Class<?>> connectorClasses) throws InterruptedException {
```

Comment:
If we want to be fancy, we can use varargs here:
```suggestion
    private void awaitMirrorMakerStart(final MirrorMaker mm, final SourceAndTarget sourceAndTarget, final <Class<?>>... connectorClasses) throws InterruptedException {
```

Reply:
I think I'll keep it as List for now as `CONNECTOR_CLASSES` is defined as `List<Class<?>>`

Reply:
Since the latest commit stopped using this new variant, maybe we can just remove it altogether?